tensorflow/compiler/mlir/hlo/tests/memref_view_alloc.mlir

```diff
@@ -0,0 +1,28 @@
+// RUN: mlir-hlo-opt --memref-view-to-alloc --split-input-file %s | FileCheck %s
+
+// Expected behavior: This code transforms the memref view to an alloc.
+// CHECK-LABEL: func @viewToAlloc
+ func @viewToAlloc(%arg0: memref<4xi8>){
+    %c0 = constant 0 : index
+    %0 = memref.view %arg0[%c0][] : memref<4xi8> to memref<i32>
+    return
+}
+
+// CHECK-SAME: %[[ARG0:.*]]: {{.*}}
```

Comment:
Why do you need to capture ARG0, C0 and V0 if they are not ever used?  I would replace
```
// CHECK-SAME: %[[ARG0:.*]]: {{.*}}
// CHECK-NEXT: %[[C0:.*]] = constant 0 : index
// CHECK-NEXT: %[[V0:.*]] = memref.alloc() : memref<i32>
```
with
```
// CHECK: memref.alloc() : memref<i32>
```

The same goes for the second test case.

Reply:
Sure, we can do this.